### PR TITLE
No jitter on startup

### PR DESCRIPTION
**Pull request by netoneko**

### CODE REVIEW

File: boyar/main/main.go [hunk 1]
```diff
@@ -104,27 +104,32 @@ func execute(flags *flags) {
...
-				reloadTimeDelay := cfg.ReloadTimeDelay(flags.maxReloadTimeDelay)
-				fmt.Println(fmt.Sprintf("INFO: waiting for %s to reload the configuration", reloadTimeDelay))
-				<-time.After(reloadTimeDelay)
-
-				ctx, cancel := context.WithTimeout(context.Background(), flags.timeout)
-				defer cancel()
-
-				if err := boyar.FullFlow(ctx, cfg, configCache); err != nil {
-					fmt.Println(time.Now(), "ERROR:", err)
+			for i := 0; true; i++ {
```
Comment (electricmonk) [inline boyar/main/main.go]: maybe:
```
for first := true; ; first = false {
}
```

File: boyar/main/main.go [hunk 2]
```diff
@@ -104,27 +104,32 @@ func execute(flags *flags) {
...
+				cfg, err := config.GetConfiguration(flags.configUrl, flags.ethereumEndpoint, flags.topologyContractAddress, flags.keyPairConfigPath)
+				if err != nil {
+					fmt.Println(time.Now(), "ERROR:", fmt.Errorf("could not generate configuration: %s", err))
+				} else {
+					// skip delay when provisioning for the first time when the node goes up
+					if i > 0 {
+						reloadTimeDelay := cfg.ReloadTimeDelay(flags.maxReloadTimeDelay)
+						fmt.Println(fmt.Sprintf("INFO: waiting for %s to apply new configuration", reloadTimeDelay))
+						<-time.After(reloadTimeDelay)
```
Comment (electricmonk) [inline boyar/main/main.go]: also select on `ctx.Done()` so that sleeps of a few minutes won't delay termination

Comment (netoneko) [reply boyar/main/main.go]: This loop is not supposed to be terminated until the node is alive.

File: boyar/main/main.go (hunk 1 above)
Comment (netoneko) [reply boyar/main/main.go]: Nice!

**Pull request merged**